masjid_util.py

```diff
@@ -0,0 +1,64 @@
+"""Masjid Utility functions."""
+
+_MASJID_METADATA = {
```

Comment:
maybe make a note that its keyed off our entity names - so if the entity name changes, then we'll need to update this map